_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -53,9 +53,23 @@ Once EOL is reached, you will need to upgrade to a newer version.
 
 _TBA: to be announced_
 
-## Foot notes
+## Protocol versioning
 
-Bitcoin Core software versioning is similar, but not identical to [SemVer][]. Bitcoin Core implements a full stack of wallet, full node, networking and consensus rule validation. However, versioning of the Bitcoin system is much more complex; there are block versions, transaction versions, p2p versions, and consensus rules are all versioned separately, and the consensus rule. [SemVer][] is not designed to handle this complexity and while Bitcoin Core adheres in the most part to the spirit of [SemVer][], there are notable exceptions, especially for consensus rules proposals.
+The description above only describes Bitcoin Core software releases. Many other parts of the Bitcoin system contain their own versions.  A few examples:
+
+- Every **block** and **transaction** contains a version number.
+- The **P2P network protocol** uses version numbers to allow nodes to announce what features they support.
```

Comment:
This is pedantry. We know nodes can lie about the services they support but that's out of scope of this document.